modules/addRemove.js

```diff
@@ -0,0 +1,24 @@
+const bookTitle = document.querySelector('#bookTitle');
+const bookAuthor = document.querySelector('#bookAuthor');
+const bookList = document.querySelector('#bookList');
+
+export const addBooks = (Books) => {
+  bookList.innerHTML = '';
+  for (let i = 0; i < Books.length; i += 1) {
+    bookList.innerHTML += `
```

Comment:
- [ ] It seems like you have not added yet [MIT license](https://github.com/johnie261/Awesome-BooksES6/tree/branch#:~:text=This%20project%20is%20MIT%20licensed.) to your project. Kindly add and make sure it's connected to your readme properly. Thank you!